### ISSUE TITLE
New tool: Mastodon Redirect

### ISSUE THREAD

**Issue by PhilC813** [duyfken/alternative-front-ends#3]
### What platform does this front-end run on?

Mobile (Android/iOS)

### Front-end Name

Mastodon Redirect

### Front-end For

Mastodon/Tools

### Front-end Repository

https://github.com/zacharee/MastodonRedirect

### Main/Official Instance

N/A

**Comment by PhilC813** [comment]
The following are also from the same developer, and included within the same repo:
- **Lemmy Redirect**
- **PeerTube Redirect**

(I thought that making separate issues would just be confusing)

------

One of the side effects of fediverse services is that due to their decentralized nature, each instance has a different URL. As a result, most mobile clients of these services, whether official or third-party, don't have all possible URLs added as supported links; there can be hundreds of them, if not thousands. So when you tap to open an instance URL, your preferred mobile client won't open up in the same way that you have expected it to work with apps like Facebook, Twitter and others. 

These apps from Zachary Wander ("zacharee" on GitHub) are cleverly fixing this problem by fetching all possible insurance URLs from [Fediverse Observer](https://fediverse.observer/), adding all of them as supported links to itself at the Android level with "one click" through tools like [Shizuku](https://github.com/RikkaApps/Shizuku), and redirects to your mobile client of choice. 

In other words, really useful tools to make your experience better with alternative front-ends of fediverse services. But now that I think about it, those are also useful for first-party apps like the official Mastodon app, so the relevance of adding them to the list is of subjective matter.

**Comment by duyfken** [comment]
Added with https://github.com/duyfken/alternative-front-ends/commit/f63b7347338351c4a1e390f7821bde6f9652976a
